src/ray/gcs/tables.cc

```diff
@@ -415,6 +415,10 @@ const ClientTableDataT &ClientTable::GetClient(const ClientID &client_id) const
   }
 }
 
+const std::unordered_map<ClientID, ClientTableDataT> &ClientTable::GetAllClients() const {
+  return client_cache_;
```

Comment:
This also contains

```
client_cache_[ClientID::nil()] = nil_client;
```

the caller has to know about this, so it should be documented in the method documentation.

Reply:
Added the document to .h file.